api/krusty/remoteload_test.go

```diff
@@ -97,11 +101,45 @@ func runResourceTests(t *testing.T, cases map[string]*remoteResourceCase) {
 			if savedTest.local && !isLocalEnv(req) {
 				t.SkipNow()
 			}
+			configureGitSSHCommand(t)
 			testRemoteResource(req, test)
 		})
 	}
 }
 
+func configureGitSSHCommand(t *testing.T) {
+	t.Helper()
+
+	// This contains a read-only Deploy Key for the kustomize repo.
+	node, err := yaml.ReadFile("testdata/repo_read_only_ssh_key.yaml")
+	if err != nil {
```

Comment:
nit: this and similar below could be shortened using `require.NoError(t, err)`

Reply:
Done